model/src/main/scala/za/co/absa/atum/model/dto/package.scala

```diff
@@ -17,15 +17,18 @@
 package za.co.absa.atum.model
 
 
-import io.circe.generic.semiauto._
 import io.circe._
 
 package object dto {
   type PartitioningDTO = Seq[PartitionDTO]
   type AdditionalDataDTO = Map[String, Option[String]]
 
+  // Todo. This implicit definition should not be defined here, so it is to be addressed in PR#221
```

Comment:
I think the PRs actually can have different numbers than the issues. But anyway I will be taking the 221 ticket and my very next one so let's not worry about correct wording here.